addons/web/static/src/views/fields/properties/property_definition_selection.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="web.PropertyDefinitionSelection" owl="1">
+        <div class="d-flex flex-column o_field_property_selection" t-ref="propertyDefinitionSelection">
+            <div class="w-100">
+                <div
+                    t-foreach="optionsVisible" t-as="option" t-key="option[0]"
+                    class="w-100 card d-flex flex-row align-items-center justify-content-between my-2 px-3 rounded o_field_property_selection_option">
+                    <span t-if="props.readonly" t-out="option[1]"/>
+                    <input
+                        t-else=""
+                        class="o_input"
+                        type="text"
+                        placeholder="Enter your option..."
+                        t-att-value="option[1]"
+                        t-on-keypress="(event) => this.onEnterOption(event, option_index)"
+                        t-on-change="(event) => this.onChangeOption(event, option_index)"
+                        t-on-blur="(event) => this.onBlurOption(event, option_index)"
+                        t-on-keydown="(event) => this.onKeyDownOption(event, option_index)"
+                    />
+                    <button
+                        class="btn btn-link text-secondary p-0 m-0"
+                        t-att-title="option[0] === props.value ? 'Remove default' : 'Set as Default'"
```

Comment:
Will those be properly translated? I'm never sure if we can write t-att-title and have them translated or not.